.github/CODEOWNERS

```diff
@@ -1,12 +1,11 @@
 # Order is important. The last matching pattern has the most precedence.
 # Protected files
-**/*webpack*    @benjaminhobbs @jamesdonoh @pjlee11 @sareh @hindsc52 @dr3 @jblaut
-/src/server/    @benjaminhobbs @jamesdonoh @pjlee11 @sareh @hindsc52 @dr3 @jblaut
-/src/client.js  @benjaminhobbs @jamesdonoh @pjlee11 @sareh @hindsc52 @dr3 @jblaut
-/envConfig/     @benjaminhobbs @jamesdonoh @pjlee11 @sareh @hindsc52 @dr3 @jblaut
-/scripts/       @benjaminhobbs @jamesdonoh @pjlee11 @sareh @hindsc52 @dr3 @jblaut
-package.json    @benjaminhobbs @jamesdonoh @pjlee11 @sareh @hindsc52 @dr3 @jblaut
-.travis.yml     @benjaminhobbs @jamesdonoh @pjlee11 @sareh @hindsc52 @dr3 @jblaut
-Jenkinsfile     @benjaminhobbs @jamesdonoh @pjlee11 @sareh @hindsc52 @dr3 @jblaut
-/cypress/       @benjaminhobbs @jamesdonoh @pjlee11 @sareh @hindsc52 @dr3 @jblaut
+**/*webpack*    @benjaminhobbs @jamesdonoh @sareh @hindsc52 @dr3 @jblaut @jroebu14
+/src/    @benjaminhobbs @jamesdonoh @sareh @hindsc52 @dr3 @jblaut @jroebu14
```

Comment:
Can you make the names line up please :L

Reply:
Done